docs/introduction-to-smart-contracts.rst

```diff
@@ -315,7 +319,7 @@ Transactions
 
 A transaction is a message that is sent from one account to another
 account (which might be the same or the special zero-account, see below).
-It can include binary data (its payload) and Ether.
+It can include binary data (its so-called payload) and Ether.
```

Comment:
Why 'so-called'? The docs don't use that phrase anywhere else, it's a fairly well-understood term.

Reply:
If "so-called" is used, it implies that this defines the technical term. Something being well-understood is not enough, I would say.

Reply:
@chriseth I get your point, how about "called a payload", sounds friendlier I think.